app/javascript/packs/application.js

```diff
@@ -7,6 +7,10 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import "bootstrap"
+import "../stylesheets/application"
```

Comment:
propose to rename folder `app/javascript` to `app/webpacker` due to style management

P.S. `# config/webpacker.yml`
```
  source_path: app/webpacker
```

Reply:
don't include styles into the main javascript pack
webpacker provides a specific pack tag for stylesheets: `stylesheet_pack_tag`